qiskit/circuit/quantumcircuit.py

```diff
@@ -3386,6 +3386,90 @@ def depth(
 
         return max(op_stack)
 
+    def depth_2q(self, include_directives: bool = False) -> int:
+        """Returns the depth in terms of two-qubit quantum gates, excluding directives by default.
+
+        Args:
+            include_directives (bool): Whether to include directives such as barriers
+
+        Returns:
+            int: Quantum circuit depth in terms of two-qubit gates
+        """
+        if not include_directives:
+            return self.depth(
+                lambda x: not getattr(x.operation, "_directive", False) and x.qubits == 2
+            )
+        else:
+            return self.depth(lambda x: x.qubits == 2)
+
+    def depth_nq(self, n: int, include_directives: bool = False) -> int:
+        """Returns the depth in terms of quantum gates with at least n-qubits,
```

Comment:
The key phrase seems to be _at least_.  And for this reason, `depth_nq` with `n=2` is _not_ the same thing as `depth_2q`.  This is potentially confusing.

Reply:
Maybe have just this one with default `n = 2` and drop `depth_2q`